corehq/apps/accounting/usage.py

```diff
@@ -39,3 +40,8 @@ def _get_sms_usage(self):
             date_sent__gte=self.start_date,
             date_sent__lt=self.end_date + datetime.timedelta(days=1),
         ).count()
+
+    def _get_web_user_usage(self):
+        web_user_in_account = set()
+        [web_user_in_account.add(id) for id in WebUser.ids_by_domain(self.domain)]
```

Comment:
Using a list comprehension here is not necessary since it constructs and then immediately discards the list of `None` values. I think it would make more sense to build the set like this:
```suggestion
        web_user_in_account = set(WebUser.ids_by_domain(self.domain))
```
It might also work to simply use the result of `WebUser.ids_by_domain(self.domain)` rather than constructing a set with that result, assuming there are not duplicates in that result.

Reply:
Thanks @millerdev! And sorry for the late reply... I think what you suggested is great and have already committed your suggestion... I should realized it myself.